auth/passport.js

```diff
@@ -30,6 +30,7 @@ const findUser = ( email, password ) => {
 const strategy = new LocalStrategy( OPTIONS, ( email, password, done ) => {
   findUser( email, password)
     .then( user => {
+      console.log("---->", user)
```

Comment:
Can you please check out this file and re-push? (Checkout should remove this line and put it in a state that will not register as changed.)